src/main/java/seedu/manager/command/MenuCommand.java

```diff
@@ -14,8 +14,8 @@ public class MenuCommand extends Command {
             add -p PARTICIPANT -e EVENT: Add a participant to an event.
             view -e EVENT: View the list of participants of an event.
             remove -p PARTICIPANT -e EVENT: Remove a participant from an event.
-            mark -e EVENT -s STATUS: Mark an event as done or not done""";
-
+            mark -e EVENT -s STATUS: Mark an event as done or not done.
+            sort -by name/time/priority.""";
```

Comment:
I think you should capitalise the placeholder term, and note what the command does beside it, in line with how the other commands are formatted in this message?

Reply:
I think name/time/priority is not placeholder here, the user can only select one of them to be the keyword of sorting.

Reply:
If that's the case, then I think you could write the line as "sort -by KEYWORD: Sorts events by name/time/priority"?

Reply:
Sounds good. I will modify it later.

Reply:
Okay, I think it looks good now!